pkg/shp/params/params.go

```diff
@@ -0,0 +1,65 @@
+package params
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/pflag"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/dynamic"
+)
+
+// ShipwrightParams is a place for Shipwright CLI to store its runtime parameters
+// including configured dynamic client and global flags
+type ShipwrightParams struct {
```

Comment:
As a naming suggestion, we should be as straightforward as `Parameters` here.

Reply:
fixed